lib/searchkick/query.rb

```diff
@@ -460,6 +465,12 @@ def prepare
         payload[:from] = offset
       end
 
+      # scroll
+      if scroll
```

Comment:
Let's raise an `ArgumentError` if invalid parameter combinations are passed instead of silently failing.